docs/devoir/01.md

```diff
@@ -2,4 +2,250 @@
 sidebar_position: 1
 ---
 
-# Devoir 1
+# Devoir 1 - Rustybox
+
+## Informations générales
+:::warning
+Date limite: **26 mars, 23:59**
+Points: **2 points** de la note finale
+Lien: Devoir 1
+Télécharger en retard: **1 points / jour** (maximum 4 jours)
+:::
+
+## Connaissances évaluées
+* Utiliser le langage Rust
+* Comprendre le mode de fonctionnement de la ligne de commande
+
+## Règles
+1. Le devoir doit contenir un fichier nommé Readme.md avec des explications concernant la manière choisie pour résoudre le devoir (-0.1p)
+2. Le devoir doit être implémenté en Rust et seulement en utilisant des fonctions de la bibliothèque standard Rust. Toute autre implémentation conduira à l'annulation du devoir.(**0 points**)
+ :::note
+ Exception: Vous pouvez utiliser la bibliotheque [chrono](https://docs.rs/chrono/latest/chrono/) pour l'affichage de date et heure.
+ :::
+
+ ## Copiage
+ Le devoir est individuel. Toute tentative de copier entraînera **0p** pour les devoirs. On utilisera un système automatisé pour détecter le copiage. Si on a des doutes, on va vous adresser des questions supplémentaires concernant le devoir.
+
+ ## Questions
+ Si vous avez des questions concernant le devoir, posez-les en publiant un issue sur le github https://github.com/UPB-FILS-SdE2/questions avec le format [busybox] "le titre de votre question" . Vous aurez besoin d'un compte github pour publier des questions.
+
+  :::danger
+  **NE PUBLIEZ PAS DE CODE SOURCE**. Cela est considéré comme copiage et vous aller recevoir **0p** pour le devoir.
+  :::
+
+  Si vous voulez recevoir un e-mail lorsque des problèmes sont signalés ou lorsqu'il y a de nouveaux messages, accédez au site [github repository](https://github.com/UPB-FILS-SdE2/questions) et cliquez sur Watch.
+
+  ## Rustybox
+  Le but de ce devoir est d’implémenter un utilitaire capable d'exécuter des commandes de type Linux bash.
+
+  Pour résoudre le devoir, vous allez réaliser un program en Rust qui reçoit comme arguments la commande qu'on désire d'exécuter, suivie par ses paramètres. Le program exécutera la commande et après il finira son exécution.
+
+  **Exemple**
+  ```bash
+    ./rustybox cp file folder
+  ```
+
+  :::note
+   Vous pouvez trouver les tests au chemin /home/codespace/devoir-1-tests dans votre GitHub Codespace.
+  :::
```

Comment:
It will be on a repo, we clone this only when running the tests. I suggest adding the link to the repo.